CHANGELOG.md

```diff
@@ -4,6 +4,14 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.39.1] - 2025-03-06
+
+This a bug-fix release that reverts the usage of polkadot-sdk umbrella crate which was causing issues with
+for example an increased number of dependencies in Cargo.lock, see [#1925](https://github.com/paritytech/subxt/issues/1925) for further information.
+
+### Fixed
+- Remove usage of polkadot-sdk umbrella crate ([#1926](https://github.com/paritytech/subxt/pull/1926))
+
```

Comment:
Should we cherry-pick https://github.com/paritytech/subxt/pull/1943 as well?

Reply:
I would say that can wait easily enough until we do a proper release next week, so that this is just 100% a patch release with no feature additions :)